signer/src/api/new_block.rs

```diff
@@ -155,7 +158,7 @@ pub async fn new_block_handler(state: State<ApiState<impl Context>>, body: Strin
                     .map(|x| created_withdrawals.push(x))
             }
             Ok(RegistryEvent::KeyRotation(event)) => {
-                handle_key_rotation(&api.ctx, event, tx_info.txid.into()).await
+                handle_key_rotation(&api.ctx, event, StacksTxId::from(tx_info.txid.0)).await
```

Comment:
We should use local types for key rotation events and withdrawal create events as well. And we should implement a conversion impl from the sbtc `StacksTxid` to the signer's `StacksTxId`.

Reply:
The local `KeyRotationEvent` was used here, which is good.